components/src/core/components/Calendar/CalendarDropdown.vue

```diff
@@ -43,9 +43,21 @@ export default defineComponent({
   },
 
   methods: {
+    toggleSubmenu() {
+      this.isActive = !this.isActive;
+      if (this.isActive) {
+        this.openSubmenu();
+      }
+    },
     openSubmenu() {
       this.isActive = true;
       this.$refs.dropdownSelector.focus();
+      this.$nextTick(() => {
+        const selectedItem = document.getElementsByClassName('--selected');
```

Comment:
Shouldn't we scope this to calendar dropdowns

Reply:
scoped using ".oxd-calendar-selector"